asab/api/log.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import datetime
+from asab.log import LOG_NOTICE
+
+
+class APIHandler(logging.Handler):
+
+	def __init__(self, level=logging.NOTSET, storage=None):
+		super().__init__(level=level)
+
+		self.level = level
```

Comment:
`self.Level` or `self._level` (aka proper member naming)

Reply:
I also believe that level is stored in the parent class (`logging.Handler`) so no need for other

Reply:
Documentation is missing ... since I have no idea whay "storage / buffer" means (and why is has two different names).
Also it feels as unnecessary API param ... it will always be a list, no reason to allow ppl to change that.

Reply:
Changed the whole section, should be more readable now. I will add documentation in the code if you want, I just have not seen it a lot in the code elsewhere...